src/locales/en.json

```diff
@@ -467,6 +468,8 @@
             "label_1": "Validator Node's Private Key",
             "description_1": "Please enter the Node's Static Private Key (you can receive one by calling admin.getNodeSigner on your node)",
             "label_3": "P-Chain address your validator node will be assigned to",
+            "label_4": "Are you sure you want to initiate a transaction? This requires signatures from other wallets in the multisignature wallet and cannot be undone",
```

Comment:
"Are you sure you want to initiate a transaction? This requires signatures from other wallets in the multi-signature wallet and if the threshold is not reached it cannot be undone for 2 weeks."